Mathlib/CategoryTheory/Action/Concrete.lean

```diff
@@ -132,12 +132,12 @@ def toEndHom [N.Normal] : G →* End (G ⧸ₐ N) where
   map_one' := by
     apply Action.hom_ext
     ext (x : G ⧸ N)
-    induction' x using Quotient.inductionOn with x
-    simp
+    obtain ⟨x⟩ := x
```

Comment:
This uses the wrong constructor, which is why the rfl appeared. Please change it back.

Reply:
Done.